pkg/parse/run.go

```diff
@@ -584,6 +592,48 @@ func (r *reconciler) startAsyncStatusUpdates(ctx context.Context) <-chan struct{
 	return doneCh
 }
 
+// unblockHydration adds the image digest into the ready-to-render file under the
+// shared directory between reconciler and hydration-controller to inform the
+// hydration-controller to proceed rendering
+func unblockHydration(commit string, r Reconciler) status.Error {
+	readyToRenderFile := r.Options().ReconcilerSignalsDir.Join(cmpath.RelativeSlash(hydrate.ReadyToRenderFile)).OSPath()
+	klog.Infof("writing commit %s to ready to render file %s", commit, readyToRenderFile)
```

Comment:
nit: `writing commit %s to ready to render file` -> `signaling the hydration-controller that commit %s is ready to render`